bpf/deserialization_to_bpf_map/deserialization_to_bpf_map.c

```diff
@@ -19,9 +19,15 @@
 #include "deserialization_to_bpf_map.h"
 #include "../../config/kmesh_marcos_def.h"
 
-#define LOG_ERR(fmt, args...)  printf(fmt, ##args)
-#define LOG_WARN(fmt, args...) printf(fmt, ##args)
-#define LOG_INFO(fmt, args...) printf(fmt, ##args)
+#define PRINTF(fmt, args...)                                                                                           \
+    do {                                                                                                               \
+        printf(fmt, ##args);                                                                                           \
+        fflush(stdout);                                                                                                \
```

Comment:
why do we need this

Reply:
without fflush we cannot see the log.
the c printf uses a buffer.

I met this when i am debugging.